terraform/projects/app-ecs-services/main.tf

```diff
@@ -14,7 +14,7 @@ variable "aws_region" {
 variable "dev_environment" {
   type        = "string"
   description = "Boolean flag for development environments"
-  default     = "true"
+  default     = "false"
```

Comment:
I can't think of any particular issues with this but I'd be interested to know if you had a particular reason for making this change?

Reply:
Basically because all of the other variables have prod as standard, figured they should probably be the same